src/applications/disability-benefits/686c-674/containers/IntroductionPage.jsx

```diff
@@ -1,37 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import { focusElement } from 'platform/utilities/ui';
-import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import { hasSession } from 'platform/user/profile/utilities';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import { verifyVaFileNumber } from '../actions';
+import { IntroductionPageHeader } from '../components/IntroductionPageHeader';
+import { IntroductionPageFormProcess } from '../components/IntroductionPageFormProcess';
+import {
+  VerifiedAlert,
+  VaFileNumberMissingAlert,
+  ServerErrorAlert,
+} from '../config/helpers';
+import { isServerError } from '../config/utilities';
 
-const VerifiedAlert = (
-  <div>
-    <div className="usa-alert usa-alert-info schemaform-sip-alert">
-      <div className="usa-alert-body">
-        <strong>Note:</strong> Since you’re signed in to your account and your
-        account is verified, we can prefill part of your application based on
-        your account details. You can also save your form in progress and come
-        back later to finish filling it out.
-      </div>
-    </div>
-    <br />
-  </div>
-);
+const alertClasses =
+  'vads-u-padding-y--2p5 vads-u-padding-right--4 vads-u-padding-left--2';
 class IntroductionPage extends React.Component {
   componentDidMount() {
+    if (hasSession()) {
```

Comment:
I wanted to prevent the verifyVaFileNumber action from firing, unless the user is authenticated.

Reply:
Not a blocker but do we know if this will always load before the component mounts?

Reply:
Yeah I think this loads before component mounts.